mathics/builtin/numbers/integer.py

```diff
@@ -464,6 +484,8 @@ def apply(self, n, base, evaluation, nr_elements=None):
 
 class IntegerDigits(_IntBaseBuiltin):
```

Comment:
This class is repeated, see above

Reply:
This was here for a while. Eventually, we should try to merge both.

Reply:
3edfde29 merges the two.